src/tests/functional/shared_test_classes/src/single_layer/prior_box.cpp

```diff
@@ -90,8 +90,6 @@ void PriorBoxLayerTest::SetUp() {
         shape_of_2,
         attributes);
 
-    ov::pass::disable_constant_folding(priorBox);
```

Comment:
As talked offline, this change will be reverted

Reply:
Done. Thanks!